### PR TITLE
Provide better guidance for HCO on CRC

### PR DESCRIPTION
**Pull request by tiraboschi**
Provide better guidance for new users
that want to easilyand quickly try HCO on CRC.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by machadovilaca — COMMENTED** [review]
My only concern is that maybe we are adding a big tool-specific section to the README. Personally, I would prefer to see you create a new file the in `docs/` folder with this guide and link it in the README.

File: README.md [hunk 1]
```diff
@@ -88,6 +88,76 @@ EOF
...
+- 64 GB of storage space
+- nested virtualization support on your laptop
+
+You can size your CRC instance with:
+```bash
+$ crc config set disk-size 64
+$ crc config set memory 20480
+```
+
+Monitoring can be optionally enable with:
```
Comment (dharmit) [inline README.md:114]:
```suggestion
Monitoring can be optionally enabled with:
```

File: README.md [hunk 2]
```diff
@@ -88,6 +88,76 @@ EOF
...
+No other manual actions are required on the CRC VM in order to be able to store and execute nested VMs inside the CRC VM.
+
+You can switch from OCP to OKD with:
+```bash
+$ crc config set preset okd
+```
+
+### Enabling nested virtualization in KVM
+Nested virtualization allows you to run a virtual machine (VM) inside another VM while still using hardware acceleration from the host.
+CRC runs as a VM, so in order to run Kubevirt on CRC to execute VMs with great performances you have to enable it.
```
Comment (dharmit) [inline README.md:129]:
```suggestion
CRC runs as a VM, so in order to run Kubevirt on CRC to execute VMs, you have to enable it.
```

File: README.md [hunk 3]
```diff
@@ -88,6 +88,76 @@ EOF
...
+
+#### Checking if nested virtualization is supported
+For Intel processors, check the /sys/module/kvm_intel/parameters/nested file. For AMD processors, check the /sys/module/kvm_amd/parameters/nested file. If you see 1 or Y, nested virtualization is supported.
```
Comment (dharmit) [inline README.md:132]:
```suggestion
For Intel processors, check the `/sys/module/kvm_intel/parameters/nested` file. For AMD processors, check the `/sys/module/kvm_amd/parameters/nested` file. If you see 1 or Y, nested virtualization is supported.
```

File: README.md [hunk 4]
```diff
@@ -88,6 +88,76 @@ EOF
...
+$ sudo modprobe kvm_intel nested=1
+```
+Nested virtualization is enabled until the host is rebooted. To enable it permanently, add `options kvm_intel nested=1` to /etc/modprobe.d/kvm.conf.
+
+To enable nested virtualization for AMD processors:
+```bash
+$ sudo modprobe -r kvm_amd
+$ sudo modprobe kvm_amd nested=1
+```
+Nested virtualization is enabled until the host is rebooted. To enable it permanently, add `options kvm_amd nested=1` to /etc/modprobe.d/kvm.conf.
```
Comment (dharmit) [inline README.md:148]:
```suggestion
Nested virtualization is enabled until the host is rebooted. To enable it permanently, add `options kvm_amd nested=1` to `/etc/modprobe.d/kvm.conf`.
```

**Review by dharmit — CHANGES_REQUESTED** [review]
Probably a few nits, but I think they are necessary in the docs.

**Pull request closed without merging**